src/Utilities/Utilities.jl

```diff
@@ -501,11 +521,25 @@ function linerange(text, from)
     return lines > 0 ? (from:(from + lines + 1)) : (from:from)
 end
 
-function format_line(range::AbstractRange)
-    top = format_line(first(range))
-    return length(range) <= 1 ? top : string(top, '-', format_line(last(range)))
+struct LineRangeFormatting
+    prefix::String
+    separator::String
+
+    function LineRangeFormatting(host::RepoHost)
+        if host == RepoBitbucket
+            new("", ":")
+        else
+            # default is github-style
+            new("L", "-")
+        end
+    end
+end
+
+function format_line(range::Range, format::LineRangeFormatting)
```

Comment:
`Range` is apparently deprecated. We should stick to `AbstractRange` I think.

```
WARNING: importing deprecated binding Base.Range into Utilities.
WARNING: Base.Range is deprecated, use Base.AbstractRange{T} where T instead.
  likely near /home/travis/.julia/v0.7/Documenter/src/Utilities/Utilities.jl:538
```

Reply:
Thanks, but this is for v0.7, right? In v0.6 we still have Range. What's the usual way you deal with these changes?

Reply:
[Compat](https://github.com/JuliaLang/Compat.jl) provides `AbstractRange` on 0.6. Compat is the standard way of supporting multiple Julia versions at the same time.

Reply:
Thanks, I didn't know about it, that's great. I've just finished addressing all comments and pushed.